src/courses/mappers/06.md

```diff
@@ -0,0 +1,309 @@
+- `summary`: Summary of security service export (e.g., the STIG header).
+- `license`: Copyright license.
+- `copyright`: Copyright holder.
+- `copyright_email`: Copyright holder's email.
+- `supports`: Supported platform targets.
+- `attributes`: Inputs/attributes used in scan.
+- `groups`: Set of descriptions for the control groups (e.g., control IDs).
+- `controls`: Controls substructure (see below).
+- `status`: Status of profile (typically 'loaded').
+
+#### Example
+
+Here's an example of a <i>profiles</i> section from a converted GoSec data file:
+
+::: details OHDF Converted GoSec Data
+```json
+"profiles": [
+  {
+    "name": "Gosec scanner",
+    "title": "gosec",
+    "version": "dev",
+    "supports": [],
+    "attributes": [],
+    "groups": [],
+    "status": "loaded",
+    "controls": [
+      ...
+    ],
+    "sha256": "d0506f4b7715bf8b1cd81a8a87ab8efcce41ebeb2b5ec5fcfb23d3bdd136f48c"
+  }
+]
+```
+:::
+
+### 2. <i>**Controls**</i>
+
+::: details Controls Schema
+```json
+controls: [
+  0: {
+    id,
+    title,
+    desc,
+    descriptions,
+    impact,
+    refs,
+    tags,
+    code,
+    source_location,
+    results
+  },
+  ... // More items may exist if there are multiple controls reported per profile
+]
+```
+:::
+
+- `id`: ID of control, used for sorting; should be unique for each unique control.
+- `title`: Title of control.
+- `desc`: Description of the control.
+- `descriptions`: Additional descriptions; usually 'check' and 'fix' text for control.
+- `impact`: Security severity of control. On a scale of 0.1 to 1. This metric is aligned with the [Common Vulnerability Scoring System (CVSS)](https://www.first.org/cvss/).
+- `refs`: References to external control documentation.
+- `tags`: Control tags; typically correlate to existing vulnerability/weakness database (e.g., NIST, CVE, CWE).
+- `code`: Control source code for code preservation.
+- `source_location`: Location of control within source code.
+- `results`: Results substructure (see below).
+
+#### Example
+
+Here's an example of a <i>controls</i> section from a converted GoSec data file:
+
+::: details OHDF Converted GoSec Data
+```json
+"controls": [
+  {
+    "tags": {
+      "nist": ["SI-10"],
+      "cwe": {
+        "id": "22",
+        "url": "https://cwe.mitre.org/data/definitions/22.html"
+      },
+      "nosec": "",
+      "suppressions": "",
+      "severity": "MEDIUM",
+      "confidence": "HIGH"
+    },
+    "refs": [],
+    "source_location": {},
+    "title": "Potential file inclusion via variable",
+    "id": "G304",
+    "desc": "",
+    "impact": 0.5,
+    "results": [
+      ...
+    ]
+  }
+],
+```
+:::
+
+### 3. <i>**Results**</i>
+
+::: details Results Schema
+```json
+results: [
+  0: {
+    status,
+    code_desc,
+    message,
+    run_time,
+    start_time
+  },
+  ... // More items may exist if there are multiple results tested per control
+]
+```
+:::
+
+- `status`: Pass/fail status of test (other statuses exist but are rare).
```

Comment:
another reminder to not discount skip/error.  don't forget to describe 'skip_message', 'backtrace', etc.

Reply:
Addressed.